pkg/render/compliance.go

```diff
@@ -85,6 +87,14 @@ var (
 	ComplianceReporterSourceEntityRule    = networkpolicy.CreateSourceEntityRule(ComplianceNamespace, ComplianceReporterName)
 )
 
+// Register secret/certs that need Server and Client Key usage
+func init() {
+	certkeyusage.SetCertKeyUsage(ComplianceServerCertSecret, []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth})
```

Comment:
When considering the init function, what made you decide to do it in the render file, rather than the controller?

Reply:
Mainly because that is where the secret name is defined.